pyproject.toml

```diff
@@ -95,3 +81,45 @@ torchseg = ["py.typed"]
 
 [tool.setuptools.packages.find]
 include = ["torchseg*"]
+
+[tool.ruff]
+exclude = [
+    # Extra files
+    "data/",
+    "images/",
+    "logo/",
+    "logs/",
+    "output/",
+
+    # Docs
+    "docs/src/",
+
+    # Spack
+    ".spack-env/",
+
+    # Python
+    "build/",
+    "dist/",
+    ".cache/",
+    ".mypy_cache/",
+    ".pytest_cache/",
+    "__pycache__/",
+    "*.egg-info/",
+
+    # Git
+    ".git/",
+    ".github/",
+]
+
+line-length = 88
+indent-width = 4
+target-version = "py311"
```

Comment:
I _think_ this is usually the minimum supported version

Reply:
Ahh okay good to know

Reply:
By default, it grabs this from `requires-python`, so no need to override: https://docs.astral.sh/ruff/settings/#target-version